src/format_ext.hpp

```diff
@@ -23,11 +23,30 @@ struct singular_prefix {
 	{
 	}
 
-	void write(fmt::Writer &w) const;
-
 };
 
 //
 // Formatting support for fmtlib
 //
-void format_arg(fmt::BasicFormatter<char> &formatter, const char *&format_str, const singular_prefix &sp);
+std::string format_as(const singular_prefix &sp);
+
+// Class to simplify migration off deprecated fmt::MemoryWriter.
+// My goal here was to minimize the diff necessary.
+// This class should probably be removed and fmt:: (or std::format) used directly.
```

Comment:
Agreed, though I don't know if std::format supports what's needed -- I can't remember if any non-trivial functionality from fmt is being used, but I don't think so.

Reply:
No, only fmt::BasicFormatter was used which doesn't exist in neither std::format nor new fmt::format - if you wish, I can remove the new class in next patch.